templates/data/opensearch/managed.html

```diff
@@ -207,9 +207,14 @@ <h2>Our process</h2>
           <div class="p-equal-height-row__col">
             <div class="p-equal-height-row__item">
               <div class="p-image-container is-highlighted u-hide--small u-hide--medium">
-                <img width="320"
-                     src="https://assets.ubuntu.com/v1/7a451504-Rectangle%203.png"
-                     alt="" />
+                {{ image(url="https://assets.ubuntu.com/v1/7a451504-Rectangle%203.png",
+                                alt="",
+                                width="852",
+                                height="1278",
+                                hi_def=True,
+                                sizes="320px",
```

Comment:
A few of these in this file are missing the image class 
```suggestion
                                sizes="320px",
                                attrs={"class": "p-image-container__image"},
```